src/SwedbankPay.Sdk.Infrastructure/OperationList.cs

```diff
@@ -0,0 +1,12 @@
+﻿using SwedbankPay.Sdk.Common;
+using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk
+{
+    public class OperationList : List<HttpOperation>, IOperationList
+    {
+        public OperationList()
```

Comment:
Codacy found an issue: [Remove this redundant constructor.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)